modules/widgets/wordpress-post-widget.php

```diff
@@ -7,19 +7,65 @@
  * Author URI: http://automattic.com
  * License: GPL2
  */
+
+/**
+ * Disable direct access/execution to/of the widget code.
+ */
+if (!defined('ABSPATH')) {
+	exit;
+}
+
 add_action( 'widgets_init', 'jetpack_display_posts_widget' );
 function jetpack_display_posts_widget() {
-	 register_widget( 'Jetpack_Display_Posts_Widget' );
+	register_widget( 'Jetpack_Display_Posts_Widget' );
 }
 
+
+/**
+ * Cron tasks
+ */
+
+/**
+ * Add a 10 minute interval to cron intervals.
+ */
+add_filter( 'cron_schedules', 'jetpack_display_posts_widget_cron_intervals' );
+function jetpack_display_posts_widget_cron_intervals() {
```

Comment:
All previously set custom intervals would have been rewritten if left as such. 

I updated the method (34df2dfd43e26d520e5a09c8b441913a921bedfc) to work as expected and not overwrite the schedules.